package.json

```diff
@@ -23,31 +23,31 @@
   "author": "GitHub",
   "license": "MIT",
   "dependencies": {
-    "@actions/cache": "https://gitpkg.now.sh/whywaita/toolkit/packages/cache?d423a21",
-    "@actions/core": "^1.2.6",
-    "@actions/exec": "^1.0.1",
-    "@actions/io": "^1.1.0",
+    "@actions/cache": "https://gitpkg.now.sh/dan-blanchard/actions-toolkit-s3/packages/cache?e16f854",
```

Comment:
I had merged [actions-toolkit-s3](https://github.com/whywaita/actions-toolkit-s3/pull/10). Please update package.json.

Reply:
Okay, I've got that set in 6d956d0